pkg/beacon/node.go

```diff
@@ -132,7 +129,7 @@ func (n *node) JoinDKGIfEligible(
 
 		err = broadcastChannel.SetFilter(membershipValidator.IsInGroup)
 		if err != nil {
-			logger.Errorf(
+			dkgLogger.Errorf(
 				"could not set filter for channel [%v]: [%v]",
 				broadcastChannel.Name(),
 				err,
```

Comment:
Can we pass `dkgLogger` to `dkg.ExecuteDKG` at line 149? That would put all DKG logs in the right context.

Reply:
Can we handle that in a separate PR? This change requires an update of the parameters passed down the way in DKG.

Reply:
https://github.com/keep-network/keep-core/issues/3314

Reply:
Addressed in https://github.com/keep-network/keep-core/pull/3323.